2020_0602 The missing piece towards any scale geospatial analytics/The missing piece towards any scale geospatial analytics.md

```diff
@@ -63,32 +63,32 @@ Compared with GeoPandas and PySAL, GeoSpark and GeoMesa adopt very different API
 
 ![](geoseries_speedup.png)
 
-In terms of interaction/exploration on large datasets, Arctern provides rendering methods that can render data (as heatmap, choropleth, etc.) with a volume of 10M in seconds. 
+In terms of interaction/exploration on large datasets, Arctern provides rendering methods that can render data (such as heatmap, choropleth, etc.) with a volume of 10M in seconds. 
 
-We are also conducting experimental GPU acceleration for spatial-temporal data analysis and rendering. By now Arctern provides six GPU-accelerated rendering methods and 8 spatial-relation operations, which outperform their CPU-based counterparts with up to 36x speed up.
+We are also conducting experimental GPU acceleration for spatial-temporal data analysis and rendering. By now Arctern provides six GPU-accelerated rendering methods and eight spatial-relation operations, which outperform their CPU-based counterparts with up to 36x speed up.
 
-In the next few versions, our team will focus on:
+In the next few releases, our team will focus on:
 
-* Developing the distributed version of GeoSeries. Our first distributed implementation of GeoDataFrame/GeoSeries will be based on Spark. It is developed in sync with Spark 3.0 since its preview release. Spark's supports on GPU scheduling and column-based processing is highly in line with our idea of high-performance spatial-temporal data processing. Besides, the introduced Koalas interface offers a promising option for implementing consistent GeoDataFrame/GeoSeries interfaces.
+* Developing a distributed version of GeoSeries. Our first distributed implementation of GeoDataFrame/GeoSeries will be based on Spark. It is developed in sync with Spark 3.0 since its preview release. Spark's supports on GPU scheduling and column-based processing is highly in line with our idea of high-performance spatial-temporal data processing. Besides, the introduced Koalas interface offers a promising option for implementing consistent GeoDataFrame/GeoSeries interfaces.
 * Enriching our spatial-temporal algorithm sets. We will concentrate on KNN search and trajectory analysis in the project's early stages. 
 
 ### About the Authors
 
-Rentong Guo is a Director of R&D at ZILLIZ. He received the Ph.D. degree in Computer Software and Theory from Huazhong University of Science and Technology (HUST), China. He spends his career participating in and leading works focused on database, cache system, distributed system, heterogeneous computing.
+Rentong Guo is a Director of R&D at ZILLIZ. He received a Ph.D. degree in Computer Software and Theory from Huazhong University of Science and Technology (HUST), China. He spends his career participating in and leading projects in the area of database, cache system, distributed system, and heterogeneous computing.
```

Comment:
He received a Ph.D. degree 
=> 
He received his Ph.D. degree

Reply:
Either way is fine.